pkg/app/server/service/webservice/service.proto

```diff
@@ -435,18 +435,16 @@ message ListAPIKeysResponse {
 
 message GetInsightDataRequest {
     model.InsightMetricsKind metrics_kind = 1 [(validate.rules).enum.defined_only = true];
-    model.InsightStep step = 2 [(validate.rules).enum.defined_only = true];
-    int64 range_from = 3 [(validate.rules).int64.gt = 0];
-    int64 data_point_count = 4 [(validate.rules).int64.gt = 0];
-    string application_id = 5;
+    int64 range_from = 2 [(validate.rules).int64.gt = 0];
+    int64 day_count = 3 [(validate.rules).int64.gt = 0];
```

Comment:
How about changing this field to `range_to` to be unified?

Reply:
Thank you!
At first I had set it to range_to, but after seeing #1187, I changed it to day_count. This is because dealing time is complicated.

Reply:
It was difficult because we have many UNIT types (DAILY, WEEKLY, ...) before.
But now only Daily is supported and I think it could be simple. 🤔

Reply:
I got your point. I'll change it to use `range_to`. Thank you!